crt_portal/cts_forms/templates/forms/complaint_view/show/attachments.html

```diff
@@ -14,11 +14,25 @@
     <input type="hidden" value="{{ return_url_args }}" name="next" id="next" />
     <input type="hidden" value="{{ index }}" name="index" id="index" />
     <input type="hidden" value="{{ data.pk }}" name="report" id="report_id" />
-    {% for attachment in attachments %}
-      <p>{{ attachment.filename }}</p>
-    {% endfor %}
+        <table class="usa-table usa-table--borderless related-reports width-full">
+            <tbody>
+                    <tr>
+                        <th style="text-align:left">File Name</th>
+                        <th style="text-align:left">ID</th>
+                        <th style="text-align:left">Added By</th>
+                        <th style="text-align:left">Created Date</th>
+                    </tr>
```

Comment:
We're not working from mock-ups but having this inside of <tbody> is rendering a top-border that looks like a duplicative. 
Let's move it into <thead>, remove the inline styling, and apply scope='col'.

<img width="613" alt="Screen Shot 2021-03-03 at 9 24 26 AM" src="https://user-images.githubusercontent.com/3485564/109821269-ceba6400-7c03-11eb-9094-164c9e7fa5aa.png">


I'm also in favor of dropping  the `ID` column, it's going to be identical for every row in the table.

Reply:
@Jkrzy We have made the necessary changes, please have another look. Thanks for all the input.  BTW, changing to get_absolute_url and filename to id cause test to fail.  That a small change we will update unit test soon.